syntax = "proto3";

package astra.http2;

message ClientConfig {
    uint32 connect_timeout_ms = 1;
    uint32 request_timeout_ms = 2;
    uint32 max_concurrent_streams = 3;
    uint32 initial_window_size = 4;
}
